pkg/reconciler/reconciler.go

```diff
@@ -394,6 +403,8 @@ func WithValueMapper(m values.Mapper) Option {
 //   - Irreconcilable - an error occurred during reconciliation
 func (r *Reconciler) Reconcile(req ctrl.Request) (res ctrl.Result, err error) {
 	ctx := context.Background()
```

Comment:
Question V

Reply:
See https://github.com/kubernetes-sigs/controller-runtime/issues/801

We could use `TODO` instead of `Background` and include a comment that links to that issue.

Reply:
done the pr; https://github.com/joelanford/helm-operator/pull/17